update_exporter_deb.py

```diff
@@ -0,0 +1,45 @@
+from datetime import date
+import argparse
+import socket
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+
+
+def main():
+    """ Read what packages were updated today
+    reads the contents of /var/log/dpkg.log, parses it and pushes the
+    information to a prometheus push gateway.
+    """
+    parser = argparse.ArgumentParser(
+        description='Send updated packages to prometheus push gateway.')
+    parser.add_argument('url', metavar='URL', type=str, default='', nargs='?',
+                        help='url for prometheus push gateway')
+
+    args = parser.parse_args()
+
+    if args.url == '':
+        print('No push gateway url given, skipping...')
+        exit()
+
+    with open('/var/log/dpkg.log', 'r') as f:
+        lines = f.readlines()
```

Comment:
Consider restructuring finding the relevant lines of the log file to something like

```
today = date.today().strftime("%Y-%m-%d")
installed = []
for line in f:
    if 'upgrade' in line and today in line:
        installed.append(line)
```

That should keep the memory usage of the script low if the dpkg log grows big for some reason.

Reply:
If I re-use this loop for building the metrics, there's no actual need for an additional array.

Reply:
well yeah, I realized that the additional array is unnecessary after writing this comment :)